nuxt.config.js

```diff
@@ -13,57 +13,46 @@ module.exports = {
   },
 
   /*
-  ** CSS
-  */
+   ** CSS
+   */
   css: [
     'normalize.css',
     '@/assets/css/main.scss',
     'element-ui/lib/theme-chalk/index.css'
   ],
 
   /*
-  ** Plugins
-  */
+   ** Plugins
+   */
   plugins: [{ src: '~/plugins/element-ui.js' }],
 
   /*
-  ** Modules
-  */
-  modules: ['@nuxtjs/axios', '@nuxtjs/proxy'],
+   ** Nuxt.js dev-modules
+   */
+  devModules: [
+    // Doc: https://github.com/nuxt-community/eslint-module
+    '@nuxtjs/eslint-module'
+  ],
 
   /*
-  ** Axios config
-  */
-  axios: {
-    baseURL: '/'
-  },
+   ** Modules
+   */
+  modules: ['@nuxtjs/axios'],
 
   /*
-  ** Proxy config
-  */
-  proxy: {
```

Comment:
Actually I wanted to keep a proxy for localhost:3000. Netlify Dev serves by default the app at both 3000 and 8888. But I guess it's not really needed: if a developer wants to test the functions, he should visit the app on localhost:8888.